packages/apps/postgres/values.yaml

```
@@ -87,11 +87,11 @@ bootstrap:
   recoveryTime: ""
   oldName: ""
 
-## @param resources Resources
+## @param resources Explicit CPU and memory configuration for each Postgres replica. When left empty, the preset defined in `resourcesPreset` is applied.
 resources: {}
  # resources:
  #   cpu: 4000m
  #   memory: 4Gi
  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)